findContact/FindContactContainer.js

```diff
@@ -88,7 +88,7 @@ class FindContactContainer extends React.Component {
       const catFilterConfig = filterConfig.find(group => group.name === 'categories');
       const oldValuesLength = catFilterConfig.values.length;
 
-      catFilterConfig.values = categories.map(rec => ({ name: rec.value, cql: rec.id }));
+      catFilterConfig.values = categories.map(rec => ({ name: rec.value, displayName: rec.value, cql: rec.id }));
```

Comment:
could you please add more info how does it help?

Reply:
You're right it doesn't help. Added prop `operator` to `filterConfig`